src/components/CodeEditor/CodeEditor.svelte

```diff
@@ -6,12 +6,19 @@
   import cssWorker from "monaco-editor/esm/vs/language/css/css.worker?worker";
   import htmlWorker from "monaco-editor/esm/vs/language/html/html.worker?worker";
   import tsWorker from "monaco-editor/esm/vs/language/typescript/ts.worker?worker";
+  import "iconify-icon";
+
+  export let handleExpand: Function;
+
   let code: string = "";
 
   let divEl: HTMLDivElement = null;
   let editor: monaco.editor.IStandaloneCodeEditor;
   let Monaco;
 
+  let languages = ["JavaScript", "Java", "Python"]
```

Comment:
We'll fetch these from a server in the next iteration

Reply:
yeah